qiskit_experiments/library/randomized_benchmarking/rb_experiment.py

```diff
@@ -121,12 +122,23 @@ def _default_experiment_options(cls) -> Options:
                 used to initialize ``numpy.random.default_rng`` when generating circuits.
                 The ``default_rng`` will be initialized with this seed value everytime
                 :meth:`circuits` is called.
+            gate_error_ratio (Union[str, bool, Dict[str, float]]): The assumption of error ratio
```

Comment:
The str and bool are special cases that can be mentioned if doc string.
```suggestion
            gate_error_ratio (Optional[Dict[str, float]]): The assumption of error ratio
```
Wording reads a bit odd and could be clearer. This should describe what the arg should look like when its a dict, which is not clear to me from reading this description. Something like:

Optional, a dict with gate name keys and error ratio values used when calculating EPG from the estimated EPC. The default value will use standard gate error ratios give in <ref, maybe to separate section in API docs?>. If set to False  EPG will not be calculated.

Note, maybe we should use None instead of False as the skip option value. You could also set this as the default so that Analysis will skip calculated EPG by default if its not set (and then set it to a concrete "default" value in the experiment). This is similar to what StateTomography does for fidelity calculations with the "target" analysis option. By default in the Analysis class it is None and won't be computed, but also by default the Experiment class will automatically calculate a default target and set that value during its execution.

Reply:
Done in https://github.com/Qiskit/qiskit-experiments/pull/762/commits/676047b854b881603db69331faec48a63de79944

The default value is set to `None` rather than `"default"` which looks much cleaner. Skipping EPG computation by default is breaking API change and some people may monitor EPGs rather than EPC.